interface/patient_file/summary/stats.php

```diff
@@ -430,7 +430,7 @@ function load_location( location ) {
     $widgetTitle = xl('Prescription');
     $widgetButtonLabel = xl("Edit");
     $widgetButtonLink = $GLOBALS['webroot'] . "/interface/patient_file/summary/rx_frameset.php";
-    $widgetButtonClass = "iframe rx_modal";
+    $widgetButtonClass = " iframe rx_modal";
```

Comment:
Is there a reason why you added a space here?

Reply:
Yes this is because when this variable was used in a function it coincides with another causing the button to be white.

Reply:
> Is there a reason why you added a space here?

OK testing.